index.html

```diff
@@ -43,5 +45,195 @@ <h1>Hey There. I'm Carlyle</h1>
+            </div>
+            <div class="cardtags">
+              <ul class="tags">
+                <li class="tag htag cardtagbg"><a href="#">html</a></li>
+                <li class="tag bootag cardtagbg"><a href="#">bootstrap</a></li>
+                <li class="tag rubytag cardtagbg"><a href="#">Ruby</a></li>
+              </ul>
+            </div>
+            <div class="fullwidth">
+              <button type="button" class="probtn1">See Project</button>
+            </div>
+          </div>
+        </div>
+        <div class="card card6">
+          <div class="card-content">
+            <div class="cardtitle">
+              <p class="ct-text">Profesional Art Printing Data</p>
+            </div>
+            <div class="cardagraph">
+              <p class="cp-text">A daily selection of privately personalized reads; no accounts or sign-ups required. Due to my wider area, I decided to add more dummy text of my own making so as to appreciate the design more.</p>
+            </div>
+            <div class="cardtags">
+              <ul class="tags">
+                <li class="tag htag cardtagbg"><a href="#">html</a></li>
+                <li class="tag bootag cardtagbg"><a href="#">bootstrap</a></li>
+                <li class="tag rubytag cardtagbg"><a href="#">Ruby</a></li>
+              </ul>
+            </div>
+            <div class="fullwidth">
+              <button type="button" class="probtn1">See Project</button>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+  <section class="info">
+    <div class="about">
+      <h2 class="am-title">About Me</h2>
+      <p class="am-description">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+      <button type="button" class="am-btn">Get My Resume</button>
+      <span class="divider"></span>
+    </div>
+    <div class="expertise">
+        <div class="chialangs">
+          <div class="diamond"></div>
+          <div class="langnames">
+            <div class="languages"><p class="langtext">Languages</p></div>
+            <ul class="langlist">
+              <li class="lang">Javascript</li>
+              <li class="lang">Ruby</li>
+              <li class="lang">Html</li>
+              <li class="lang">Css</li>
+            </ul>
+          </div>
+        </div>
+        <div class="chiaframes">
+          <div class="square"></div>
+          <div class="framenames">
+            <div class="frameworks"><p class="frametext">Frameworks</p></div>
+            <ul class="framelist">
+              <li class="fram">Bootstrap</li>
+              <li class="fram">Ruby on Rails</li>
+              <li class="fram">RSPec</li>
+              <li class="fram">CapyBara</li>
+              <li class="fram">Selenium</li>
+            </ul>
```

Comment:
- [ ] [Required] Your alignment here is good for sure. The only issue is that you did not use the same color as it is in the Figma template. Try to also maintain the same margins and paddings with the Figma template.
## Yours
![image](https://user-images.githubusercontent.com/45668340/148662029-fcf53d88-b761-4ebb-a8b3-b963c8793eaf.png)

## Figma's
![image](https://user-images.githubusercontent.com/45668340/148662060-695471a5-70cf-438d-93dc-57a0b387c380.png)